coralproject/talk
# add nodemon config to prevent server restart while developing front end

## What does this PR do?

there's a delay when updating the front-end code, as the entire app restarts when you change a component. This will save some time while developing.

## How do I test this PR?

- `npm run dev-start`
- change something on the front-end. app should not restart
- change a server file. app should restart
